tests/unit/shared/utils/normalize-table-test.js

```diff
@@ -0,0 +1,37 @@
+const { join } = require('path')
+const test = require('tape')
+const cheerio = require('cheerio')
```

Comment:
Hey @ryanblock , I only need to get cheerio to here (a test file). Everything else works without changing npm anything. If I delete the tests I can revert all package.json files. Please let me know what the best way to test is without bringing extra dependencies along for the ride to production.

Reply:
Do we deploy dev dependencies?  I don't think we'll deploy anything in `tests` to prod.

Reply:
Put it in dev deps; technically it can go in deps but then folks won't be able to keep track of why/which. tldr if it's a dependency for tests, put it in dev deps.